tools/proofers/for_mentors.php

```diff
@@ -181,16 +187,26 @@ function output_project_details($mentored_round, $projectid, $nameofwork, $autho
 
 // -------------------------------------------------------------------
 
-function get_beginner_projects_in_state($state)
+function get_beginner_projects_in_state($state, $mentored_round)
 {
+    $mentored_round_detail = $mentored_round . '.proj_done';
+
     $sql = sprintf("
-        SELECT projectid, nameofwork, authorsname
-        FROM projects
+        SELECT
+            projects.projectid,
+            projects.nameofwork,
+            projects.authorsname,
+            MAX(project_events.timestamp) AS t_left_mentee_round
+        FROM projects LEFT OUTER JOIN project_events USING (projectid)
         WHERE
-            difficulty = 'BEGINNER' AND
-            state='%s'
+            difficulty = 'BEGINNER'
+            AND project_events.event_type = 'transition'
+            AND project_events.details2 = '%s'
+            AND projects.state='%s'
```

Comment:
Consider improving the prior version and put spaces around `=`.

Reply:
Done. I also took the liberty of looking for other instances and added spaces in the couple I found.